tsconfig.json

```diff
@@ -3,7 +3,7 @@
 		"target": "es5",
 		"module": "commonjs",
 		"moduleResolution": "node",
-		"allowJs": true,
+		"declaration": true,
 		"allowSyntheticDefaultImports": true,
```

Comment:
allowJs is not longer required because now that we have typings we don't need requires in our TS.